buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -132,16 +199,17 @@ def print_projects_need_to_migrate(failed_jobs_per_flag):
 def print_flags_need_to_migrate(failed_jobs_per_flag):
     # The info box printed later is above info box printed before,
     # so reverse the flag list to maintain the same order.
+    printed_flag_boxes = False
     for flag in sorted(list(failed_jobs_per_flag.keys()), reverse=True):
         jobs = failed_jobs_per_flag[flag]
         if jobs:
             github_url = INCOMPATIBLE_FLAGS[flag]
-            info_text = []
-            info_text.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
+            info_text = [f"* **{flag}** " + get_html_link_text(":github:", github_url)]
             info_text += merge_and_format_jobs(jobs.values(), "  - **{}**: {}")
             # Use flag as the context so that each flag gets a different info box.
             print_info(flag, "error", info_text)
-    if len(info_text) == 1:
```

Comment:
Note: this was a bug in the original code

Reply:
Thanks for fixing the bug!